structuredheadings/plugin.js

```diff
@@ -219,6 +219,24 @@
 
       CKEDITOR.dialog.add("selectStyle", this.path + "dialogs/selectstyle.js");
 
+      // Indent and outdent with TAB/SHIFT+TAB key
+      editor.on("key", function (evt) {
+        if (editor.mode !== "wysiwyg") {
+          return;
```

Comment:
I'm not positive to be honest, but this part was taken from the code for the IndentList plugin. I'm not sure of the exact logic behind what the wysiwyg "mode" does or doesn't do, but if the official plugin for a keyboard bind does this check I thought it best to duplicate it.